README.md

```diff
@@ -358,6 +360,30 @@ resources:
 
 To disable the model validation you can use `--disableModelValidation`.
 
+## WebSocket
+`This has experimental functionality. Please report any bugs or missing features.`
+
+serverless-offline suports running a WebSocket local endpoint.
+
+Usage in order to send messages back to clients:
+
+`POST http://localhost:{port+1}/@connections/{connectionId}`
+
+Or,
+
+```
+let endpoint=event.apiGatewayUrl;
+if (!endpoint) endpoint = event.requestContext.domainName+'/'+event.requestContext.stage;
+const apiVersion='2018-11-29';
+const apiGM=new API.ApiGatewayManagementApi({ apiVersion, endpoint });
+apiGM.postToConnection({ConnectionId, Data});
```

Comment:
Where does `API` come from?
Please lint (spacing) this and use `apiGatewayManagementApi` instead of `apiGM`

Reply:
Previously it was API=AWS||{injected API in context}.
Should be now AWS instead of API.